helloworld/lib/goodbye.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Looks like the goodbye modules still got pushed up, but I saw you did put the path names into the your .gitignore. So obviously, I'm not sure of the correct syntax to hide specific modules, I'll ask Duncan about it later. :)